packages/backend/services/db/dbLocal.js

```diff
@@ -8,13 +8,13 @@ require("dotenv").config();
 const fs = require("fs");
 const { getProp } = require("../../utils/object");
 
-console.log("using local db");
-
-const DATABASE_PATH = "./local_database/local_db.json";
-const SEED_PATH = "./local_database/seed.json";
+const DATABASE_PATH = process.env.DATABASE_PATH ? process.env.DATABASE_PATH : "./local_database/local_db.json";
+const SEED_PATH = process.env.SEED_PATH ? process.env.SEED_PATH : "./local_database/seed.json";
```

Comment:
I think we can solve this better doing something like:

```js
fs.readFileSync(path.resolve(__dirname, "../../local_database/local_db.json"));
```

So it's always relative to this file and we don't have to worry about where it's called (& avoid new env vars)